### PR TITLE
chore: do not use cdn for fonts

### LINKED ISSUE #116: Remove usage of google fonts
__What should we do?__

Do not dynamically load fonts from google fonts. Use locally cached fonts.

`ibm PLEX` can specifically be installed directly of `npm`.

__Why should we do it?__

Recently found to violate GDPR.

### PR DESCRIPTION
**Pull request by marstamm**
closes #116

### CODE REVIEW

**Comment by nikku** [comment]
My proposal is to craft a custom `CSS` to explicitly declare only the required fonts; we'll otherwise load 170kb of CSS (before minification of course) _before_ loading any font: 

image

**Comment by marstamm** [comment]
I added the custom css as part of `app.css` in 7431946fae2c67ca1f00d58e4ebcb138de6bd495

I took the required fonts from the old google CDN path.

**Comment by nikku** [comment]
Great. Added a few small adjustments (basically code cleanups) on top:

* prefix include with _ (convention)
* rename plex -> fonts
* only copy actual font data to dist (datensparsamkeit ;o))

File: Gruntfile.js [hunk 1]
```diff
@@ -175,6 +178,13 @@ module.exports = function(grunt) {
             cwd: '<%= config.bootstrap.dist %>',
             src: ['js/bootstrap.min.js'],
             dest: '<%= config.dist %>/assets'
+          },
+          // include ibm plex
+          {
+            expand: true,
+            cwd: '<%= config.plex.dist %>',
+            src: ['IBM-Plex-Sans/fonts/complete/**/*'],
```
Comment (marstamm) [inline Gruntfile.js:186]: It's missing `IBM-Plex-Mono`.

```Suggestion
            src: ['IBM-Plex-Sans/fonts/complete/**/*', 'IBM-Plex-Mono/fonts/complete/**/*'],
```

Comment (nikku) [reply Gruntfile.js:186]: My bad :).

The font was not used yet, hence I forgot to add it.

Comment (marstamm) [reply Gruntfile.js:186]: image

Indeed, it isn't... I'll look into changing the default monospace font to Plex

Comment (nikku) [reply Gruntfile.js:186]: Did it already! https://github.com/bpmn-io/bpmn.io/pull/118

**Pull request merged**